osu.Framework/Input/StateChanges/Events/JoystickAxisChangeEvent.cs

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Input.States;
+
+namespace osu.Framework.Input.StateChanges.Events
+{
+    public class JoystickAxisChangeEvent : InputStateChangeEvent
+    {
+        public readonly int Axis;
+
+        public readonly float Value;
+
+        public JoystickAxisChangeEvent(InputState state, IInput input, int axis, float value)
```

Comment:
Always use `JoystickAxis` to refer to (`int axis, float value`)
```suggestion
        public JoystickAxisChangeEvent(InputState state, IInput input, JoystickAxis axis)
```